code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -193,3 +193,34 @@
 		pulling = null
 		usr.pulledby = null
 	..()
+		
+/obj/item/wheelchair
+	name = "wheelchair"
+	desc = "A folded wheelchair that can be carried around."
+	icon = 'icons/obj/furniture.dmi'
+	icon_state = "wheelchair_folded"
+	item_state = "wheelchair"
+	w_class = ITEMSIZE_HUGE // Can't be put in backpacks. Oh well.
+
+/obj/item/wheelchair/attack_self(mob/user)
+		var/obj/structure/bed/chair/wheelchair/R = new /obj/structure/bed/chair/wheelchair(user.loc)
+		R.add_fingerprint(user)
+		R.name = src.name
+		R.desc = src.desc
+		R.color = src.color
+		qdel(src)
+
+/obj/structure/bed/chair/wheelchair/MouseDrop(over_object, src_location, over_location)
+	..()
+	if((over_object == usr && (in_range(src, usr) || usr.contents.Find(src))))
+		if(!ishuman(usr))
+			return FALSE
+		if(buckled_mob)
+			return FALSE
+		visible_message("[usr] collapses \the [src.name].")
+		var/obj/item/wheelchair/R = new(get_turf(src))
+		R.name = src.name
+		R.desc = src.desc
+		R.color = src.color
+		return
```

Comment:
You do still need to qdel the thing at the end of this, otherwise you'll be left with the `/obj/structure/bed/chair/wheelchair` after you folded it up into your hand. It just didn't need the spawn(0).